pkgs/development/libraries/science/math/libtorch/bin.nix

```diff
@@ -12,6 +12,7 @@
   fixDarwinDylibNames,
 
   cudaSupport ? config.cudaSupport,
+  cudaPackages_13,
```

Comment:
Why not using the `cudaPackages` alias?

Reply:
It's because the binary being downloaded from torch's website is the cuda 13 version. 

see https://github.com/collinarnett/nixpkgs/blob/bd8c806f7fb8d89256729c3d058915af715bf3db/pkgs/development/libraries/science/math/libtorch/prefetch.sh#L9

If there is a good pattern having mulitple cuda versions in the prefetch script, I'd be happy to implement it.